iml-agent/src/action_plugins/check_ha.rs

```diff
@@ -71,27 +71,146 @@ impl AgentInfo {
     }
 }
 
+fn _check_ha(output: &[u8]) -> Result<Vec<AgentInfo>, ImlAgentError> {
```

Comment:
Any reason we need the leading `_`. It's sort of implied it's private by the lack of the `pub` keyword.